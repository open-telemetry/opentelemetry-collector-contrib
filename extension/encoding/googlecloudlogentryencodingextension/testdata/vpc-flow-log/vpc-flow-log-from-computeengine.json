{
  "insertId": "x1ty6xe56du7",
  "jsonPayload": {
    "dest_vpc": {
      "subnetwork_region": "us-central1",
      "project_id": "test-project-id",
      "vpc_name": "default",
      "subnetwork_name": "default"
    },
    "bytes_sent": "0",
    "start_time": "2025-09-27T21:34:39.250342106Z",
    "end_time": "2025-09-27T21:34:39.250342106Z",
    "connection": {
      "dest_port": 443,
      "src_ip": "203.0.113.4",
      "src_port": 55524,
      "dest_ip": "192.0.2.4",
      "protocol": 6
    },
    "packets_sent": "16",
    "dest_instance": {
      "managed_instance_group": {
        "name": "test-mig-2",
        "zone": "us-central1-a"
      },
      "project_id": "test-project-id",
      "vm_name": "test-vm-4",
      "region": "us-central1",
      "zone": "us-central1-a"
    },
    "network_service": {
      "dscp": 32
    },
    "reporter": "DEST"
  },
  "resource": {
    "type": "gce_subnetwork",
    "labels": {
      "subnetwork_id": "1234548292813243874",
      "subnetwork_name": "default",
      "project_id": "test-project-id",
      "location": "us-central1"
    }
  },
  "timestamp": "2025-09-27T21:34:41.615575788Z",
  "logName": "projects/test-project-id/logs/compute.googleapis.com%2Fvpc_flows",
  "receiveTimestamp": "2025-09-27T21:34:41.615575788Z"
}
