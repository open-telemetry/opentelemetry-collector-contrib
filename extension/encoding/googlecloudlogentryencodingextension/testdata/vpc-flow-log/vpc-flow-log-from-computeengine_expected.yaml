resourceLogs:
  - resource:
      attributes:
        - key: gcp.project
          value:
            stringValue: test-project-id
        - key: cloud.resource_id
          value:
            stringValue: compute.googleapis.com%2Fvpc_flows
        - key: gcp.resource_type
          value:
            stringValue: gce_subnetwork
        - key: gcp.label.subnetwork_id
          value:
            stringValue: "1234548292813243874"
        - key: gcp.label.subnetwork_name
          value:
            stringValue: default
        - key: gcp.label.project_id
          value:
            stringValue: test-project-id
        - key: gcp.label.location
          value:
            stringValue: us-central1
    scopeLogs:
      - logRecords:
          - attributes:
              - key: log.record.uid
                value:
                  stringValue: x1ty6xe56du7
              - key: network.protocol.name
                value:
                  stringValue: tcp
              - key: source.address
                value:
                  stringValue: 203.0.113.4
              - key: destination.address
                value:
                  stringValue: 192.0.2.4
              - key: source.port
                value:
                  intValue: "55524"
              - key: destination.port
                value:
                  intValue: "443"
              - key: gcp.vpc.flow.reporter
                value:
                  stringValue: DEST
              - key: gcp.vpc.flow.bytes_sent
                value:
                  intValue: "0"
              - key: gcp.vpc.flow.packets_sent
                value:
                  intValue: "16"
              - key: gcp.vpc.flow.start_time
                value:
                  stringValue: "2025-09-27T21:34:39.250342106Z"
              - key: gcp.vpc.flow.end_time
                value:
                  stringValue: "2025-09-27T21:34:39.250342106Z"
              - key: gcp.vpc.flow.destination.instance.project.id
                value:
                  stringValue: test-project-id
              - key: gcp.vpc.flow.destination.instance.vm.region
                value:
                  stringValue: us-central1
              - key: gcp.vpc.flow.destination.instance.vm.name
                value:
                  stringValue: test-vm-4
              - key: gcp.vpc.flow.destination.instance.vm.zone
                value:
                  stringValue: us-central1-a
              - key: gcp.vpc.flow.destination.instance.managed_instance_group.name
                value:
                  stringValue: test-mig-2
              - key: gcp.vpc.flow.destination.instance.managed_instance_group.zone
                value:
                  stringValue: us-central1-a
              - key: gcp.vpc.flow.destination.project.id
                value:
                  stringValue: test-project-id
              - key: gcp.vpc.flow.destination.subnet.name
                value:
                  stringValue: default
              - key: gcp.vpc.flow.destination.subnet.region
                value:
                  stringValue: us-central1
              - key: gcp.vpc.flow.destination.vpc.name
                value:
                  stringValue: default
              - key: gcp.vpc.flow.network_service.dscp
                value:
                  intValue: "32"
            body: {}
            observedTimeUnixNano: "1759008881615575788"
            timeUnixNano: "1759008881615575788"
        scope:
          attributes:
            - key: encoding.format
              value:
                stringValue: gcp.vpcflow
